### PR TITLE
Update add-bitsfile.md

### PR DESCRIPTION
**Pull request by sukihanabi**
#1052 
Action taken:

Added " BITS supports throttled and asynchronously transferring large files between a client and a server using idle network bandwidth. BITS intelligently pick which transfer jobs to run, increase or decrease the rate at which files are transferred based on the amount of idle network bandwidth available. See [Adding a file to a job](https://docs.microsoft.com/windows/desktop/bits/background-intelligent-transfer-service-portal) and [About BITS](https://docs.microsoft.com/windows/desktop/bits/using-bits) to learn more. "

References:

https://social.technet.microsoft.com/Forums/office/en-US/3a3f4e4f-118a-4706-89c6-38a746ac203c/difference-between-bits-and-smb

https://answers.microsoft.com/en-us/windows/forum/windows_10-networking/bits-always-running-in-background/154371c7-a855-40af-85ff-301f0092055d

Using BITS
https://docs.microsoft.com/en-us/windows/desktop/bits/using-bits

Adding a file to a job
https://docs.microsoft.com/en-us/windows/desktop/bits/adding-files-to-a-job

https://docs.microsoft.com/en-us/windows/desktop/bits/background-intelligent-transfer-service-portal

About BITS
https://docs.microsoft.com/en-us/windows/desktop/bits/about-bits

https://docs.microsoft.com/en-us/powershell/module/bitstransfer/start-bitstransfer?view=win10-ps

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/bitstransfer/add-bitsfile.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ Add-BitsFile [-BitsJob] <BitsJob[]> [[-Destination] <String[]>] [-Source] <Strin
 The **Add-BitsFile** cmdlet adds files to a Background Intelligent Transfer Service (BITS) transfer job.
 You can specify the files to add to the BITS transfer job by name at the command prompt or in a comma-separated value (CSV) file.
 
+BITS supports throttled and asynchronously transferring large files between a client and a server using idle network bandwidth. BITS intelligently pick which transfer jobs to run, increase or decrease the rate at which files are transferred based on the amount of idle network bandwidth available. See [Adding a file to a job](https://docs.microsoft.com/windows/desktop/bits/background-intelligent-transfer-service-portal) and [About BITS](https://docs.microsoft.com/windows/desktop/bits/using-bits) to learn more.
```
Comment (JohanFreelancer9) [inline docset/windows/bitstransfer/add-bitsfile.md]:
```suggestion
BITS supports throttled and asynchronously transferring large files between a client and a server using idle network bandwidth. BITS intelligently pick which transfer jobs to run, increase or decrease the rate at which files are transferred based on the amount of idle network bandwidth available. See [Background Intelligence Transfer Service](https://docs.microsoft.com/windows/desktop/bits/background-intelligent-transfer-service-portal) and [Using BITS](https://docs.microsoft.com/windows/desktop/bits/using-bits) to learn more. 
```

**Comment by get-itips** [comment]
Hello @sukihanabi 
I think that the issue creator wants an explanation or example of passing an array of BitsJob[] and you are providing theory behind BITS technology, which is not the same. 

As the Description for the BitsJob parameter says, the cmdlet Get-BitsTransfer can return a BitsJob or an array of BitsJobs, and that is the way you can pass an array to this cmdlet.

Please try to test this behavior on your lab, for this, you will have to create more than one Bits Transfer using Start-BitsTransfer and then use Get-BitsTransfer and piping the result to this cmdlet.

What I think it will happen is it will add the file to be downloaded by every job.

**Comment by sukihanabi** [comment]
@get-itips - Thanks, that's noted, working on it.

**Comment by sukihanabi** [comment]
> Hello @sukihanabi
> I think that the issue creator wants an explanation or example of passing an array of BitsJob[] and you are providing theory behind BITS technology, which is not the same.
> As the Description for the BitsJob parameter says, the cmdlet Get-BitsTransfer can return a BitsJob or an array of BitsJobs, and that is the way you can pass an array to this cmdlet.
> Please try to test this behavior on your lab, for this, you will have to create more than one Bits Transfer using Start-BitsTransfer and then use Get-BitsTransfer and piping the result to this cmdlet.
> What I think it will happen is it will add the file to be downloaded by every job.
Hello @get-itips , added a new note on the PR. feel free to let me know for any changes. Thank you for your assistance.

**Comment by get-itips** [comment]
Looks better

**Comment by dariomws306** [comment]
I am ok with this.

**Comment by sukihanabi** [comment]
> I am ok with this.

Thank you @dariomws306 @get-itips

**Comment by tseward** [comment]
Looks good.

**Comment by e0i** [comment]
@AndreaBarr The PR can be merged. Thanks.

**Pull request merged**